src/Core/Tool/Authorizer/PostAuthorizer.php

```diff
@@ -61,7 +61,7 @@ class PostAuthorizer implements Authorizer, Permissionable
      */
     protected function getAllPrivs()
     {
-        return ['read', 'create', 'update', 'delete', 'search', 'change_status'];
+        return ['read', 'create', 'update', 'delete', 'search', 'change_status', 'change_timestamp'];
```

Comment:
Don't need this change_timestamp isn't a separate permission, just part of `update`